src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -124,6 +129,39 @@ function ReportActionItemMessageEdit(props) {
         isFocusedRef.current = isFocused;
     }, [isFocused]);
 
+    useEffect(() => {
+        if (Platform.OS === 'web' && !Browser.isMobile()) {
```

Comment:
we should not use such checks - you need to create a separate files for handle web and other platforms